internal/resources/metal/organization/datasource_schema.go

```diff
@@ -0,0 +1,84 @@
+package organization
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+var dataSourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"name": schema.StringAttribute{
+			Description: "The name of the Organization",
+			Required:    true,
+		},
+		"organization_id": schema.StringAttribute{
+			Description: "The UUID of the organization resource",
+			Optional:    true,
+			Computed:    true,
+		},
+		"description": schema.StringAttribute{
+			Description: "Description string",
+			Optional:    true,
+		},
+		"website": schema.StringAttribute{
+			Description: "Website link",
+			Optional:    true,
+		},
+		"twitter": schema.StringAttribute{
+			Description: "Twitter handle",
+			Optional:    true,
+		},
+		"logo": schema.StringAttribute{
+			Description: "Logo URL",
+			Optional:    true,
+		},
+		"project_ids": schema.ListNestedAttribute{
+			Computed: true,
+			NestedObject: schema.NestedAttributeObject{
+				Attributes: map[string]schema.Attribute{
+					"id": schema.StringAttribute{
+						Computed: true,
+					},
+				},
+			},
+		},
```

Comment:
This is a different case because the element type is an object. First you need to change the schema. You are using the resource `address` schema instead of the datasource schema. All the fields were computed https://github.com/equinix/terraform-provider-equinix/blob/450917b0e9018ebf302231a5fd1606b1889e3a78/equinix/data_source_metal_organization.go#L72-L96

You will usually need to define a `ListNestedBlock` but it is different for computed blocks (all fields in the nested object are computed). Use this as an example: https://github.com/equinix/terraform-provider-equinix/blob/450917b0e9018ebf302231a5fd1606b1889e3a78/internal/resources/metal/connection/datasource_schema.go#L94-L99

for `address` in the resource schema you will need to define a `ListNestedBlock`  in the blocks section, because there you have some editable fields (required/optional). It would be something like:

```
var dataSourceSchema = schema.Schema{
    Attributes: map[string]schema.Attribute{
        // YOUR ATTRIBUTES
    },
    Blocks: map[string]schema.Block{
        "address": schema.ListNestedBlock{
             Description: "Address information block",
             CustomType:  fwtypes.NewListNestedObjectTypeOf[PortModel](ctx),
             NestedObject: schema.NestedBlockObject{
                 Attributes: map[string]schema.Attribute{
                     "address": schema.StringAttribute{ ... },
                     "city": schema.StringAttribute{ ... },
                  },
	},
```